### ISSUE TITLE
Custom events in old Android

### ISSUE THREAD

**Issue by paulxb** [twbs/ratchet#744]
I had a problem with and Android 4.1.2 phone where it wasn't raising the toggle custom event. Apparently because the WebView doesn't support CustomEvent.
I found a piece of code that solves the problem:
```
/*
 * Polyfill for adding CustomEvent
 * see : https://developer.mozilla.org/fr/docs/Web/API/CustomEvent
 */
if (!window.CustomEvent) { // Create only if it doesn't exist
    (function () {
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        };

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
}
```

Source: http://stackoverflow.com/a/25626257/197167

It might be worth adding this to the source to help others? (I'm not really sure of the best place for it!)

**Comment by Johann-S** [comment]
They already had a PR for this : https://github.com/twbs/ratchet/pull/675 but we still waiting for the merge of this PR

**Comment by mdo** [comment]
Appears to have been resolved with #748?

**Comment by Johann-S** [comment]
Yes exactly
